site/docs/contribute.md

```diff
@@ -45,6 +45,16 @@ The Iceberg community prefers to receive contributions as [Github pull requests]
 * If a PR is related to an issue, adding `Closes #1234` in the PR description will automatically close the issue and helps keep the project clean
 * If a PR is posted for visibility and isn't necessarily ready for review or merging, be sure to convert the PR to a draft
 
+### Merging Pull Requests
+
+Most pull requests can be merged once a single committer is satisfied with the code in the PR. Before merging all review comments should be addressed either by making changes or agreeing the request is out of scope for the PR. For larger changes or additions to public APIs committers will wait at least 24 hours before merging to ensure there is no additional feedback from members of the community. As in all ASF governed projects committers are expected to act in the best interest of the project. If a committer feels there might be a conflict of interest with a pull request they review, they are encouraged to ask for another committer to review the pull request.
+
+Requesting changes on a PR indicates a reviewer believes the PR has merit but still needs issues addressed before merging. If a reviewer believes the change should not be merged at all and there is nothing the author could do to address the reviewers concerns (e.g. the reviewer feels the change should go through the [Iceberg improvement proposal](#apache-iceberg-improvement-proposals)), the reviewer should explicitly state this on the PR. In the rare event that a PR author and reviewers cannot come to a consensus on a PR, the disagreement should be raised to the developer mailing list for further discussion, and if necessary, a vote. In this context a reviewer is anyone leaving comments on the PR including contributors, committer and PMC members.
```

Comment:
I disagree with this because votes should not be used to settle disagreements. Votes are used to confirm consensus and as a last chance for dissenting opinions. This is really important in communities like Iceberg because we want to keep focused on the best technical solution, not forming a majority.

Reply:
I removed the language about a vote.